docs/CHANGELOG.asciidoc

```diff
@@ -77,6 +78,354 @@ Review important information about the {kib} 8.x releases.
+* Calculate the latest metadata lookback based on the calculated history delay ({kibana-pull}191324[#191324]).
+* Remove dedicated language setting ({kibana-pull}190983[#190983]).
+* Change latest metric to use @timestamp ({kibana-pull}190417[#190417]).
+* Prevent initial error when adding filters ({kibana-pull}190214[#190214]).
+* Display error message when failing to enable machine learning anomaly detection in Inventory ({kibana-pull}189627[#189627]).
+* Convert route validation to Zod ({kibana-pull}188691[#188691]).
+* Fixes functions table height in asset details view profiling tab ({kibana-pull}188650[#188650]).
+* Adds four decimal places float validation for transaction_sample_rate ({kibana-pull}188555[#188555]).
+* Centralize data fetching and better control of when data can be refreshed ({kibana-pull}187736[#187736]).
+* Fixes heatmap on monitor detail/history page for very large doc counts ({kibana-pull}184177[#184177]).
+* Adds settings to serverless allowlist ({kibana-pull}190098[#190098]).
+* Set missing group to false by default and show checkbox value in disable mode ({kibana-pull}188402[#188402]).
+Elastic Search solution::
+* Fixes an issue with the {ref}/es-connectors-network-drive.html[Network Drive connector] where advanced configuration fields were not displayed for CSV file role mappings with `Drive Type: Linux` selected.
+Elastic Security solution::
+For the Elastic Security 8.16.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Kibana platform::
+* Fixes an issue causing a wrong date to show in the header of a report when generated from relative date ({kibana-pull}197027[#197027]).
+* Fixes an issue where the Created and Updated timestamps for Dashboards were ignoring the default timezone settings in Advanced settings. ({kibana-pull}196977[#196977]).
+* Fixes an issue causing searches including a colon `:` character to show inaccurate results ({kibana-pull}190464[#190464]).
+Kibana security::
+* Fixes an issue where an LLM was likely to generate invalid processors containing array access in Automatic Import ({kibana-pull}196207[#196207]).
+Machine Learning::
+* File upload: fixes PDF character count limit ({kibana-pull}197333[#197333]).
+* Data Drift: Updates brush positions on window resize fix ({kibana-pull}196830[#196830]).
+* AIOps: Fixes issue where some queries cause filters to not be applied ({kibana-pull}196585[#196585]).
+* Transforms: Limits the data grid result window ({kibana-pull}196510[#196510]).
+* Fixes Anomaly Swim Lane Embeddable not updating properly on query change ({kibana-pull}195090[#195090]).
+* Hides ES|QL based saved searches in ML & Transforms ({kibana-pull}195084[#195084]).
+* Fixes query for pattern analysis and change point analysis ({kibana-pull}194742[#194742]).
+* Various fixes for possible prototype pollution vulnerabilities ({kibana-pull}194529[#194529]).
```

Comment:
This one should be removed from the release notes. I amended the release note label on the PR to `release_note:skip`

Reply:
```suggestion
```